src/ports/linux/pnal_options.h.in

```diff
@@ -18,14 +18,4 @@
 
 #include <net-snmp/net-snmp-config.h>
 
-/** Priority of SNMP thread (if SCHED_FIFO is used) */
```

Comment:
This file is now empty so let's remove it and update cmakefile.

Reply:
Done